localenv/docs/peer-setup.md

```diff
@@ -418,12 +418,12 @@ pointer ID:
 Query
 
 ```
-mutation CreatePaymentPointer ($input: CreateAccountInput!) {
-  createPaymentPointer(input: $input) {
+mutation CreateWalletAddress ($input: CreateAccountInput!) {
```

Comment:
Just above a reference to payment pointer

Reply:
Is the reference still there? I don't see it in this file.

Reply:
L415: "execute the following command to create a payment
pointer ID:"